core/api/src/graphql/public/mutations.ts

```diff
@@ -113,6 +114,8 @@ export const mutationFields = {
 
       callbackEndpointAdd: CallbackEndpointAdd,
       callbackEndpointDelete: CallbackEndpointDelete,
+
+      csvReportGenerate: CSVReportGenerate,
```

Comment:
This would only make sense if we were creating a scheduled csv and another query to receive the csv url... otherwise must not be a mutation